java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/impl/InsertPreparedStatementImpl.java

```diff
@@ -48,7 +50,8 @@ public class InsertPreparedStatementImpl implements PreparedStatement {
     private final List<Object> currentDatas;
     private final List<DataType> currentDatasType;
     private final List<Boolean> hasSet;
-    private final List<Integer> scehmaIdxs;
+    // stmt insert idx -> real table schema idx
+    private final List<Pair<Long, Integer>> schemaIdxes;
```

Comment:
the first type of pair is the same as second. set as Integer

Reply:
uint32 is long, and idx type of list is int, no the same.